website/templates/share_search.mako

```diff
@@ -6,4 +6,16 @@
 
 <%def name="javascript_bottom()">
     <script src=${"/static/public/js/share-search-page.js" | webpack_asset}></script>
+
+<script type="text/x-mathjax-config">
+   MathJax.Hub.Config({
+       tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']], processEscapes: true}
```

Comment:
What's more, arXiv itself doesn't render the text-mode macros, only the math-mode ones:

http://arxiv.org/abs/1304.5129

![screen shot 2016-04-19 at 4 37 51 pm](https://cloud.githubusercontent.com/assets/134455/14654744/250547bc-064d-11e6-84cb-f143f56b0199.png)